### PR TITLE
Issue #126: Support `NOSTEM`

### PR DESCRIPTION
**Pull request by mnunberg**
`NOSTEM` can now be added as a keyword when creating a text field.

### CODE REVIEW

File: docs/Commands.md [hunk 1]
```diff
@@ -7,7 +7,7 @@
   FT.CREATE {index} 
     [NOOFFSETS] [NOFIELDS] [NOSCOREIDX]
     [STOPWORDS {num} {stopword} ...]
-    SCHEMA {field} [TEXT [WEIGHT {weight}] | NUMERIC | GEO] [SORTABLE] ...
+    SCHEMA {field} [TEXT NOSTEM [WEIGHT {weight}] | NUMERIC | GEO] [SORTABLE] ...
```
Comment (dvirsky) [inline docs/Commands.md]: should be `[NOSTEM]`

**Comment by jrots** [comment]
for what it's worth the golang client lib generates create an index create command like this : 
`jaymetest SCHEMA hb NUMERIC ha TEXT WEIGHT 1 NOSTEM`
which results in `(error) Could not parse field spec` on latest master.

this works though: `jaymetest SCHEMA hb NUMERIC ha TEXT NOSTEM WEIGHT 1`
not sure where to report as it could be that the nostem field can be put freely in index definition.

**Comment by dvirsky** [comment]
@mnunberg this is definitely a bug, there should be no importance to the order of the arguments.

**Comment by mnunberg** [comment]
@jrots I'll start working on a solution - this is a bug, file it if I don't file one first

**Comment by jrots** [comment]
Ok done! @mnunberg

**Pull request merged**